modules/developer_manual/pages/testing/unit-testing.adoc

```diff
@@ -78,21 +79,96 @@ You can find more information in {phpunit-docs-url}[the PHPUnit documentation].
 [[running-php-unit-tests-for-owncloud-10.0]]
 === Running PHP Unit tests for ownCloud >= 10.0
 
-There are existing tests provided by ownCloud which are ready to run.
+There are existing test options provided by ownCloud.
+To run them, change into the root directory of your ownCloud installation and run `grep "make test" <(make help)` to see tests and parameters available.
 
-* Change into `webroot` and run `make help` to see tests and parameters
-available.
+You should see output similar to the below example.
 
-Testing apps
+[source,console]
+----
+make test-acceptance-api	run API acceptance tests
+make test-acceptance-cli	run CLI acceptance tests
+make test-acceptance-webui	run webUI acceptance tests
+make test-js-debug		run Javascript tests in debug mode (continuous)
+make test-js			run Javascript tests
+make test-php-phan		run PHP phan static code analyzer
+make test-php-phpstan		run PHP phpstan static code analyzer
+make test-php-style-fix		run PHP code style checks and fix any issues found
+make test-php-style		run PHP code style checks
+make test-php-unit		run all PHP tests
+make test			run all tests
+----
+
+==== Testing Apps
 
-* To run the tests for a specific app with the provided PHPUnit version,
-change into `<webroot>/apps/<appname>` and
+To run the tests for a specific app with the provided PHPUnit version: 
 
+. Change into the apps directory, in the root directory of your ownCloud installation
```

Comment:
To run unit tests for a "separate" app, you need to clone it from GitHub.
Now that we have the `apps-external` directory by default in an installation, I recommend to put the separate apps in there, and not pollute the `apps` folder. e.g.
```
cd apps-external
git clone https://github.com/owncloud/files_antivirus.git
cd files_antivirus
make test-php-unit
```